test/functional/domains_controller_test.rb

```diff
@@ -1,15 +1,11 @@
 require 'test_helper'
+require 'functional/shared/basic_rest_response_test'
```

Comment:
I'll do that, but I don't really see the point to have the module included in classes that don't use any of the methods